lib/trailer/storage/cloud_watch.rb

```diff
@@ -49,6 +39,16 @@ def flush
         retry
       end
 
+      # Queues the given hash for writing to CloudWatch.
+      #
+      # @param data [Hash] A key-value hash of trace data to write to storage.
+      def write(data)
+        messages << {
+          timestamp: (Time.now.utc.to_f.round(3) * 1000).to_i,
+          message:   data&.to_json,
```

Comment:
PS: maybe we can skip the messages without data?

Reply:
Good idea, I missed it 👍